lib/galaxy/config_watchers.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, app, start_thread=True):
             app.config,
             'watch_core_config',
             monitor_what_str='core config file',
-            event_handler_class=SimpleFileModifiedEventHandler
+            event_handler_class=EventHandler
```

Comment:
this is the default, the import and the specification is not needed.

Reply:
Done.